testdata/google/events/cloud/pubsub/v1/MessagePublishedData-binary.json

```diff
@@ -0,0 +1,8 @@
+{
+   "subscription": "projects/my-project/subscriptions/my-subscription",
```

Comment:
Indentation is inconsistent. Should be 2 spaces in all JSON files.

Reply:
Will do (although probably not this minute).